src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -81,27 +81,44 @@ typedef struct otPlatformConfig
 } otPlatformConfig;
 
 /**
- * This function performs all platform-specific initialization of OpenThread's drivers.
+ * This function performs all platform-specific initialization of OpenThread's drivers and initialize the OpenThread
+ * instance.
  *
  * @note This function is not called by the OpenThread library. Instead, the system/RTOS should call this function
  *       when initialization of OpenThread's drivers is most appropriate.
  *
  * @param[in]  aPlatformConfig  Platform configuration structure.
  *
- * @returns A pointer to the OpenThread instance.
+ * @sa otSysGetInstance
  *
  */
-otInstance *otSysInit(otPlatformConfig *aPlatformConfig);
+void otSysInit(otPlatformConfig *aPlatformConfig);
 
 /**
- * This function performs all platform-specific deinitialization for OpenThread's drivers.
+ * This function performs all platform-specific deinitialization for OpenThread's drivers and finalizes the OpenThread
+ * instance.
  *
  * @note This function is not called by the OpenThread library. Instead, the system/RTOS should call this function
  *       when deinitialization of OpenThread's drivers is most appropriate.
  *
+ * @sa otSysGetInstance
+ *
  */
 void otSysDeinit(void);
 
+/**
+ * This function gets the OpenThread instance.
```

Comment:
Maybe emphasize that the instance is a singleton.

Reply:
It's now removed.